openapi-specs/shared/schemas.yaml

```
@@ -0,0 +1,258 @@
+access:
```

```suggestion
access-item:
```
I think the access array type should be added to the shared schemas.
As is, the one access type entry per grant restriction is only defined in the RS<>Client spec:
https://github.com/interledger/open-payments/blob/23bd697583293884822c6aa9c0944ae8a13aea02/openapi-specs/AS/client/openapi.yaml#L437-L455

---

Good catch!